src/front/glsl/lex_pp.rs

```diff
@@ -0,0 +1,242 @@
+use super::{parser::Token, token::TokenMetadata, types::parse_type};
```

Comment:
does this module need to be renamed? I understand that it's rewritten, but the `pp` suffix doesn't seem to be needed if we don't have `lex.rs` any more